src/console-wrapper.js

```diff
@@ -42,11 +42,14 @@ async function testCafeRunner () {
 }
 
 if (require.main === module) {
+  console.log(`Sauce TestCafe Runner ${require(path.join(__dirname, '..', 'package.json')).version}`);
   const { runCfgPath, suiteName } = getArgs();
 
   testCafeRunner(runCfgPath, suiteName)
       // eslint-disable-next-line promise/prefer-await-to-then
-      .then((passed) => process.exit(passed ? 0 : 1))
+      .then(() => {
+          process.exit(0);
```

Comment:
Hm, in the cypress runner we still do it like `.then((passed) => process.exit(passed ? 0 : 1))`. Why is this one different?

Reply:
@alexplischke the `console-wrapper.js` file calls the method `testCafeRunner` which calls `testcafeRunner.js` as a subprocess. If the subprocess exits with `0` the method `testCafeRunner` is "resolved"; otherwise it is rejected.

Hence why it's `.then(() => process.exit(0)` and `.catch(() => process.exit(1)`